test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/ValueFormTests/FirstUpperCaseValueFormTests.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.TemplateEngine.Orchestrator.RunnableProjects.ValueForms;
+using System.Globalization;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests.ValueFormTests
+{
+    public class FirstUpperCaseValueFormTests
+    {
+        [Theory]
+        [InlineData("a", "A", null)]
+        [InlineData("no", "No", null)]
+        [InlineData("new", "New", null)]
+        [InlineData("", "", null)]
+        [InlineData(null, null, null)]
+        [InlineData("indigo", "İndigo", "tr-TR")]
+        [InlineData("ındigo", "Indigo", "tr-TR")]
+        public void FirstUpperCaseWorksAsExpected(string input, string expected, string culture)
+        {
+            if (!string.IsNullOrEmpty(culture))
+            {
+                if (culture == "invariant")
```

Comment:
Is this meant to be `null`?

Reply:
No, it is not. 
This snippet sets the culture if not null in data. 
To use invariant culture, `"invariant"` const should be used. It is not used in this test in particular, but other tests are using it.
If data is null, current culture is used. 

The only thing came to my mind, is that we should probably reset it once test is done, otherwise other tests will be executed with last culture set.

Reply:
I would rather address it in another PR, as there are other tests which do not reset culture.